openebs/.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: required
+dist: trusty
+group: edge
+services:
+  - docker
+language: go
+go:
+  - 1.8
+script:
+  - make bootstrap
+  - make deps
+  - make build
+after_success:
+  - make image
+notifications:
+  email:
+    recipients:
+    - kiran.mova@cloudbyte.com
```

Comment:
Actually, As of now we don't need `.travis.yaml` file anymore! I am removing it as it is redundant now.